packages/reporting-server/rest_server/default_config.py

```diff
@@ -0,0 +1,24 @@
+# pylint: disable=line-too-long
+config = {
+    "host": "127.0.0.1",  # ip or hostname to bind the socket to
+    "port": 8002,
+    "db_url": "sqlite://:memory:",
+    # "root_path": "/api",
+    # url that rmf-server is being served on.
+    # When being a proxy, this must be the url that rmf-server is mounted on.
+    # E.g. https://example.com/rmf/api/v1
+    "public_url": "http://localhost:8002",
+    # The directory where static files should be stored.
+    "static_directory": "static",
+    "log_level": "DEBUG",  # https://docs.python.org/3.8/library/logging.html#levels
```

Comment:
Not sure if `DEBUG` should be the default log level.

Reply:
Changed to WARN